apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -191,12 +191,11 @@ describe('Notifications Creator', function () {
 
   it('should validate form inputs', function () {
     cy.visit('/templates/create');
-    cy.getByTestId('submit-btn').click();
-
-    cy.getByTestId('title').should('have.class', 'mantine-TextInput-invalid');
     addChannel('inApp');
+    cy.getByTestId('description').type('this is a notification template description');
```

Comment:
I think you should change the order of addChannel and typing into description. I think that should fix the test